admin/models/school.js

```diff
@@ -5,9 +5,9 @@ const Schema = mongoose.Schema
 const randomGenerator = require('../lib/random-generator')
 
 const School = new Schema({
-  _id: { type: Number },
+  _id: {type: Number},
   leaCode: {
-    type: Number,
+    type: Schema.Types.Number,
```

Comment:
Why changed that from Number to Schema.Types.Number?
I was under the impression it was the same